Detectors/Calibration/SpacePoints/include/SpacePoints/Param.h

```diff
@@ -0,0 +1,121 @@
+  96.475, 97.225, 97.975, 98.725, 99.475, 100.225, 100.975, 101.725, 102.475, 103.225, 103.975, 104.725, 105.475, 106.225, 106.975,
+  107.725, 108.475, 109.225, 109.975, 110.725, 111.475, 112.225, 112.975, 113.725, 114.475, 115.225, 115.975, 116.725, 117.475, 118.225,
+  118.975, 119.725, 120.475, 121.225, 121.975, 122.725, 123.475, 124.225, 124.975, 125.725, 126.475, 127.225, 127.975, 128.725, 129.475,
+  130.225, 130.975, 131.725, 135.100, 136.100, 137.100, 138.100, 139.100, 140.100, 141.100, 142.100, 143.100, 144.100, 145.100, 146.100,
+  147.100, 148.100, 149.100, 150.100, 151.100, 152.100, 153.100, 154.100, 155.100, 156.100, 157.100, 158.100, 159.100, 160.100, 161.100,
+  162.100, 163.100, 164.100, 165.100, 166.100, 167.100, 168.100, 169.100, 170.100, 171.100, 172.100, 173.100, 174.100, 175.100, 176.100,
+  177.100, 178.100, 179.100, 180.100, 181.100, 182.100, 183.100, 184.100, 185.100, 186.100, 187.100, 188.100, 189.100, 190.100, 191.100,
+  192.100, 193.100, 194.100, 195.100, 196.100, 197.100, 198.100, 199.350, 200.850, 202.350, 203.850, 205.350, 206.850, 208.350, 209.850,
+  211.350, 212.850, 214.350, 215.850, 217.350, 218.850, 220.350, 221.850, 223.350, 224.850, 226.350, 227.850, 229.350, 230.850, 232.350,
+  233.850, 235.350, 236.850, 238.350, 239.850, 241.350, 242.850, 244.350, 245.850
+};
+static constexpr float RowDX[NPadRows] = { // distance between pad rows
+  0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750,
+  0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750,
+  0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750, 0.750,
+  1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000,
+  1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000,
+  1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000, 1.000,
+  1.500, 1.500, 1.500, 1.500, 1.500, 1.500, 1.500, 1.500, 1.500, 1.500, 1.500, 1.500, 1.500, 1.500, 1.500, 1.500, 1.500, 1.500, 1.500, 1.500, 1.500,
+  1.500, 1.500, 1.500, 1.500, 1.500, 1.500, 1.500, 1.500, 1.500, 1.500, 1.500
+};
+static constexpr float SecDPhi = 20.f * 0.01745; // 20.f * pi / 180.f
+static constexpr float MaxY2X = 0.176;           // TMath::Tan(0.5f * SecDPhi);
+static constexpr float DeadZone = 1.5f;
+static constexpr float MaxZ = 1.f;
+
+// for internal data structures
+static constexpr int ResDim = 4;
+static constexpr int VoxDim = 3;
+static constexpr int VoxHDim = 4;
```

Comment:
please add some comments about these dimensions (in aliroot, with enums, they were more explicit)

Reply:
I have added some comments for now, but I think I will also move these parameters to TrackResiduals.h.
First, I did not want that header to become very large, but I think there is no way around it.